torch/csrc/deploy/deploy.cpp

```diff
@@ -160,6 +160,14 @@ ReplicatedObj InterpreterSession::createMovable(Obj obj) {
   TORCH_CHECK(
       manager_,
       "Can only create a movable object when the session was created from an interpreter that is part of a InterpreterManager");
+  auto objIntpSession = obj.getInterpreterSessionImpl();
+
+  if (objIntpSession != impl_.get()) {
+    TORCH_CHECK(
+        false,
+        "Cannot create movable from an object that lives in different session");
```

Comment:
Can we improve the error message to specify which interpreter the object is on, and which interpreter is trying to create the movable object? I think interpreters have ids associated with them, right? That should make debugging easier.

Reply:
is it notifyIdx_ in InterpreterSession? Obj has InterpreterSessionImpl as its' field but I don't think it knows anything about its' ID. So I can write an error message with Obj's ID and InterpreterSession's notifyIdx_. But getting the obj's interpreterSession's ID seems bit tricky.

Reply:
@suo what do you think about this?